### ISSUE TITLE
Seems to be need an "user-agent" for requesting

### ISSUE THREAD

**Issue by 8zizek8** [danielpetri1/bbb-recording-exporter#90]
I get this Error:

            open_http': 401 Unauthorized (OpenURI::HTTPError)

I think the reason is the lack of user-agent in the request

**Comment by hostbbb** [comment]
its possible with a bash script to download the events.xml, and the raw recording objects locally, then run the ruby script to create the mp4.   but this only works wint 2.4,  2.5   does not work anymore with 2.6 and newer with the tldraw whiteboard.   So this script only works with older bbb recordings. if you post a sample recording can tell you what version.

**Comment by 8zizek8** [comment]
![Screenshot from 2023-12-06 01-18-09](https://github.com/danielpetri1/bbb-recording-exporter/assets/126919049/f64a489e-ebf6-46b5-81a4-64cdefd505f6)

**Comment by 8zizek8** [comment]
![Screenshot from 2023-12-07 01-24-07](https://github.com/danielpetri1/bbb-recording-exporter/assets/126919049/6f4d243e-bb71-463f-afd6-fef6d33da265)

**Comment by hostbbb** [comment]
I see your issue.....

https://conf2.anisa.co.ir/presentation/aefeeb8e34eb48cdd846993a15051c80ca4593c1-1692879978011/metadata.xml

change your path... you have the playback link  with the /playback/2.3/  so wont find the files.

**Comment by 8zizek8** [comment]
Thank you for your reply.
did you mean I should change url exactly to this?
[](https://conf2.anisa.co.ir/presentation/aefeeb8e34eb48cdd846993a15051c80ca4593c1-1692879978011/metadata.xml)

**Comment by 8zizek8** [comment]
Thank you for your reply.
did you mean I should change url exactly to this:
https://conf2.anisa.co.ir/presentation/aefeeb8e34eb48cdd846993a15051c80ca4593c1-1692879978011/metadata.xml
?

**Comment by hostbbb** [comment]
no, in the download ruby file  PATH="https://conf2.anisa.co.ir/presentation/aefeeb8e34eb48cdd846993a15051c80ca4593c1-1692879978011/#{file}"

**Comment by 8zizek8** [comment]
Thank you, I downloaded it. Now I go to the next step

**Comment by 8zizek8** [comment]
The output is complete. Thank you.
Is it possible to remove the webcam and chat in the output so that the whole page becomes the main content?
